apps/site/src/lib/styles/globals.scss

```diff
@@ -1,4 +1,25 @@
+@use "bootstrap-utils" as bootstrap;
+
+$container-padding: 8rem;
+
 .background {
 	background-image: url("~@/assets/background/anteater-head-tiling.gif");
 	background-size: 464px; // half size for 2x scaling
 }
+
+.accordion {
+	--bs-accordion-active-color: theme.$black;
+	--bs-accordion-bg: transparent;
+	--bs-accordion-active-bg: transparent;
+
+	--bs-accordion-border-color: transparent;
+	--bs-accordion-btn-focus-box-shadow: 0;
```

Comment:
Issue: Mentioned before in #35, but removing the focus box shadow inhibits accessible navigation.

Reply:
I put back the box shadow, but it doesn't look as good. Probably worth the compromise though.
<img width="732" alt="image" src="https://github.com/HackAtUCI/zothacks-site-2023/assets/28026520/bc0bb654-6af5-4795-8ebb-c3cd59f8bec2">